zappa/handler.py

```diff
@@ -129,14 +129,14 @@ def handler(self, event, context):
                 response = Response.from_app(app, environ)
 
                 # This is the object we're going to return.
-                zappa_returndict = dict()
+                # Pack the WSGI response into our special dictionary.
+                zappa_returndict = {header_name: header_value
+                                    for (header_name, header_value)
+                                    in response.headers}
 
                 if response.data:
                     zappa_returndict['Content'] = response.data
```

Comment:
If 'Content' is in response.headers, it now behaves differently than before (it will be set by response.data instead of by the header). This is probably correct, but should be double checked.